pilot/pkg/networking/core/v1alpha3/route/retry/retry.go

```diff
@@ -22,8 +22,19 @@ import (
 	route "github.com/envoyproxy/go-control-plane/envoy/api/v2/route"
 	"github.com/golang/protobuf/ptypes/wrappers"
 
+	previouspriorities "github.com/envoyproxy/go-control-plane/envoy/config/retry/previous_priorities"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/any"
 	networking "istio.io/api/networking/v1alpha3"
 	"istio.io/istio/pilot/pkg/networking/util"
+	"istio.io/pkg/log"
+)
+
+const (
+	googleApis = "type.googleapis.com/"
+
+	retryPreviousPrioritiesConfig = googleApis + "envoy.config.retry.previous_priorities.PreviousPrioritiesConfig"
```

Comment:
You do not have to build these typeURLS - You should use `util.MessageToAny`

Reply:
Done